datasets/np/mha_sanctions/crawler.py

```diff
@@ -0,0 +1,81 @@
+import csv
+from typing import Dict
+import zavod.helpers as h
+from zavod import Context
+
+DATE_FORMATS = ["%m/%d/%Y"]
+
+
+def crawl_row(context: Context, row: Dict[str, str]):
+    unsc_id = row.pop("code")
+    entity_type = row.pop("type")
+    name = row.pop("name")
+    # Split `alias` on `;` and trim any extra whitespace
+    alias = row.pop("alias").split(";")
+    birth_date = h.parse_date(row.pop("date of birth"), DATE_FORMATS)
+    birth_place = row.pop("place of birth").split(";")
+    nationality = row.pop("nationality").split(";")
+    pass_no = row.pop("passport no").split(";")
+    national_id = row.pop("national id")
+    address = row.pop("address").split(";")
+    listing_date = h.parse_date(row.pop("listed on"), DATE_FORMATS)
+    sanction_status = row.pop("sanction status")
+    remarks = row.pop("remarks")
+
+    entity = None
+    if entity_type == "INDIVIDUALS":
+        entity = context.make("Person")
+        entity.id = context.make_id(name, birth_date, unsc_id)
+        entity.add("name", name)
+        for a in alias:  # Add aliases
+            entity.add("alias", a.strip())
+        for d in birth_date:
+            entity.add("birthDate", d.strip())
+        for p in birth_place:
+            entity.add("birthPlace", p.strip())
+        for n in nationality:
+            entity.add("nationality", n.strip())
+        for p in pass_no:
+            entity.add("passportNumber", p.strip())
```

Comment:
Sorry, only spotted this now.

What could be really nice is for the passport numbers where `p.tolower().split(" number: ")` gives you two values, treat those values as `county, number` and

- put the number only in the passportNumber field - that will improve matching
- h.make_identification with the country name and number - that's more for end users making sense of the match result

![image](https://github.com/user-attachments/assets/b1942945-43fa-4efd-b0da-4b17ef016905)